kube-client/Cargo.toml

```diff
@@ -62,7 +62,7 @@ jsonpath-rust = { workspace = true, optional = true }
 tokio-util = { workspace = true, features = ["io", "codec"], optional = true }
 hyper = { workspace = true, features = ["client", "http1"], optional = true }
 hyper-util = { workspace = true, features = ["client", "client-legacy", "http1", "tokio"], optional = true }
-hyper-rustls = { workspace = true, optional = true }
+hyper-rustls = { workspace = true, features = ["http1", "logging", "native-tokio", "ring", "tls12"], optional = true }
```

Comment:
Ah. Thanks for this.
I was a bit worried about potentially having to handle [aws-lc-rs](https://github.com/rustls/hyper-rustls/blob/0d387948b62641d30631170ad994d288c6fbde98/Cargo.toml#L37C1-L37C10) but this is a nice way to avoid having to deal with that immediately.